examples/js/hello-sdl/readme.md

```diff
@@ -1,8 +1,7 @@
 ## SDL JavaScript App Startup Instructions
 1) Place the vanilla JS SDL.min.js file into this directory
 1) Request a [Manticore instance](https://smartdevicelink.com/resources/manticore/)
-1) Open two terminal sessions and `cd` both of them into `./examples/js/hello-sdl`
-1) In one terminal session, run `java -jar proxy.jar m.sdl.tools [PORT]`, where `[PORT]` is the one given to you by Manticore
-1) In the other terminal session, run `npm install` then `npm start`.
+1) Open a terminal session and `cd` into `./examples/js/hello-sdl`
+1) In the terminal session, run `npm install` then `npm start`.
 1) Your browser should automatically open and the test app should appear on your Manticore UI.
```

Comment:
There should also be a step added where it explains that you have to edit the index.html file and edit the WebSocketClientConfig arguments to use Manticore's provided websocket address.